instance/models/mixins/openedx_storage.py

```diff
@@ -147,6 +149,14 @@ def _get_swift_settings(self):
             "SWIFT_LOG_SYNC_REGION_NAME": self.swift_openstack_region,
         }
 
+    def provision_s3(self):
+        """
+        Create S3 Bucket if it doesn't exist
```

Comment:
Oddly, unlike [`create_iam_user`](https://github.com/open-craft/opencraft/pull/295/files#diff-b66ef4710ed40f1b980e3ab3a34ec812R126), running this method multiple times doesn't throw a `EntityAlreadyExists` error, so it seems like the bucket is ignored if it already exists.